### DISCUSSION TITLE
Consider develop a CodeMirror 6 adapter?

### DISCUSSION THREAD

**Discussion by alissa-tung** [loro-dev/loro#542] [General]
Hello, this project looks very nice! It provides a set of neat APIs for collaborative editing features. I noticed that there is already a ProseMirror adapter, would you consider build one for CodeMirror 6 too?

**Comment by zxch3n** [comment]
Yes, there are some reusable works for CodeMirror that we can share.

**Comment by Leeeon233** [comment]
Hi @alissa-tung , [loro-codemirror](https://github.com/loro-dev/loro-codemirror) is open-source now.
